lib-multisrc/comicaso/src/eu/kanade/tachiyomi/multisrc/comicaso/Comicaso.kt

```diff
@@ -0,0 +1,466 @@
+            "Adult",
+            "Adventure",
+            "Age Gap",
+            "Aliens",
+            "Animals",
+            "Anthology",
+            "BDSM",
+            "Beasts",
+            "Bloody",
+            "Bodyswap",
+            "Boys",
+            "Cheating/Infidelity",
+            "Childhood Friends",
+            "College life",
+            "Comedy",
+            "Contest winning",
+            "Cooking",
+            "Crime",
+            "Crossdressing",
+            "Delinquents",
+            "Demons",
+            "Drama",
+            "Dungeons",
+            "Ecchi",
+            "Emperor's daughte",
+            "Fantasy",
+            "Fetish",
+            "Fighting",
+            "Full Color",
+            "Game",
+            "Gender Bender",
+            "Genderswap",
+            "Ghosts",
+            "Girl",
+            "Girls",
+            "Gore",
+            "Harem",
+            "Hentai",
+            "Historical",
+            "Horror",
+            "Incest",
+            "Isekai",
+            "Josei(W)",
+            "Kids",
+            "Magic",
+            "Magical Girls",
+            "Manga",
+            "Manhua",
+            "Manhwa",
+            "Martial Arts",
+            "Mature",
+            "Medical",
+            "Military",
+            "Monster Girls",
+            "Monsters",
+            "Music",
+            "Mystery",
+            "Non-human",
+            "NTR",
+            "Office Workers",
+            "Omegaverse",
+            "Oneshot",
+            "Philosophical",
+            "Police",
+            "Psychological",
+            "Regression",
+            "Reincarnation",
+            "Revenge",
+            "Reverse Harem",
+            "Reverse Isekai",
+            "Romance",
+            "Royal family",
+            "Royalty",
+            "School Life",
+            "Sci-Fi",
+            "Seinen(M)",
+            "Sejarah",
+            "Shoujo ai",
+            "Shoujo(G)",
+            "Shounen ai",
+            "Shounen(B)",
+            "Showbiz",
+            "Slice of Life",
+            "SM/BDSM/SUB-DOM",
+            "Smut",
+            "Space",
+            "Sports",
+            "Super Power",
+            "Superhero",
+            "Supernatural",
+            "Survival",
+            "Thriller",
+            "Time Travel",
+            "Tower Climbing",
+            "Traditional Games",
+            "Tragedy",
+            "Transmigration",
+            "Vampires",
+            "Video Games",
+            "Villainess",
+            "Violence",
+            "Virtual Reality",
+            "Western",
+            "Wuxia",
+            "Yakuzas",
+            "Yaoi(BL)",
+            "Yuri(GL)",
+            "Zombies",
+        )
+    }
+
+    protected class StatusFilter : Filter.Select<String>(
+        "Status",
+        arrayOf("All", "Completed"),
+    )
+
+    override fun popularMangaSelector(): String = throw UnsupportedOperationException()
```

Comment:
since you are not using most of parse methods, consider inheriting from `HttpSource` directly